browser/src/react-redux/App/User/User.actions.js

```diff
@@ -1,8 +1,8 @@
-const setUser = (user) => {
-  return {
-    type: 'SET_USER',
-    user
-  }
-}
+export const setUserAction = user => ({
```

Comment:
Is there a reason you added "Action" to the variable name? I think we can just keep these as `setUser` and `resetUser`.

Reply:
We were naming the method in Signup and Login setUser as well, so wasn't liking same name.

Reply:
What wasn't like the same name? You? Prettier?

Reply:
Put it back to just resetUser and get this error 
`'resetUser' is already declared in the upper scope.eslint(no-shadow)`